moc_sprint_tools/cmd/copy_cards.py

```diff
@@ -0,0 +1,43 @@
+import click
+import github
+import logging
+
+from moc_sprint_tools.sprintman import BoardNotFoundError
+from moc_sprint_tools import defaults
+
+LOG = logging.getLogger(__name__)
+
+
+def CommaDelimitedList(val):
+    if val is None:
+        return None
+
+    return val.split(',')
+
+
+@click.command(name='copy-cards')
+@click.option('-f', '--from', 'src', metavar='BOARD_NAME')
+@click.option('-t', '--to', 'dst', metavar='BOARD_NAME')
+@click.option('-c', '--copy-columns', type=CommaDelimitedList)
+@click.option('-i', '--ignore-columns', type=CommaDelimitedList)
```

Comment:
I don't understand how we can set type to a function. Could you point me to some click documentation where that's explained?

Reply:
The `type` parameter is just a callable that takes a single argument and returns a value. So you can e.g. set `type=int`, which given a string value like `"10"` would run `int("10")` to generate the value assigned to the option.

Similarly, `CommaDelimitedList` takes a single value -- a string like `"one,two,three"` -- and transform it into a list (`["one", "two", "three"]`).

There are examples in https://click.palletsprojects.com/en/7.x/options/. The `argparse` module works pretty much the same way, and the `argparse` documentation is [explicit about this behavior](https://docs.python.org/3/library/argparse.html#type).

Reply:
Thanks! Your explanation clears it up.